stdio/file.c

```diff
@@ -170,12 +189,14 @@ int fclose(FILE *file)
 {
 	int err;
 
-	if (file == NULL)
+	if (file == NULL) {
 		return -EINVAL;
+	}
 
 	fflush(file);
-	while ((err = safe_close(file->fd)) < 0 && errno == EINTR)
+	while ((err = safe_close(file->fd)) < 0 && errno == EINTR) {
 		;
+	}
```

Comment:
A bit of the PR but, repeated condition in `safe_close()` function.
```suggestion
	err = safe_close(file->fd);
```

Reply:
Thanks, we will resolve it in the another PR